.travis.yml

```
@@ -5,7 +5,9 @@ jdk:
   - openjdk8
 
 script:
-  - sbt clean coverage +test coverageReport && sbt coverageAggregate
+  - sbt clean coverage test \
+    +play26-json-tests-sc13/test \
+    coverageReport && sbt coverageAggregate
```

```suggestion
    sbt coverageAggregate
```
https://github.com/scoverage/sbt-scoverage#notes-on-upgrading-to-version-160

---

Tight!